tools/pipelines/templates/build-npm-package.yml

```diff
@@ -85,6 +85,10 @@ parameters:
   type: string
   default: repo
 
+- name: packageManagerInstallCommand
+  type: string
+  default: 'npm ci --unsafe-perm'
```

Comment:
Could we add a brief comment explaining why we need `unsafe-perm` here?

Reply:
I don't recall why we use this. It may have been inherited from when we used Docker to build. @curtisman, do you remember?